### DISCUSSION TITLE
requestLegacyExternalStorage - Android update regards MANAGE_EXTERNAL_STORAGE and privacy

### DISCUSSION THREAD

**Discussion by MayoudP** [react-native-image-picker/react-native-image-picker#1672] [Q&A]
<img width="708" alt="Screenshot 2021-04-22 at 14 56 51" src="https://user-images.githubusercontent.com/48682041/115718155-28751a00-a37b-11eb-81cf-ca724630a2e0.png">

Hi everyone, regarding the upcoming update by the Android team core. How is it impacting this library ? One of the option would be to include `MANAGE_EXTERNAL_STORAGE` in the manifest as suggested, or else to use another native android library to handle the doc selection. 

Anyway, I'm getting a warning about this issue on my play developper console, and it need to be updated. 

Any suggestion, or more informations about this ? 

Thanks.

**Comment by billnbell** [comment]
Anyone?

**Comment by AugustoAleGon** [comment]
This is something critical for May 5th. It seems that requestLegacyExternalStorage is being ignored but what is gonna happen with the storage?

**Comment by danghung24031992** [comment]
Any solution?

**Comment by boerkey** [comment]
good question, hopefully we can get some suggestion for this.

**Comment by chr4ss1** [comment]
update to the latest version 3.5.0, get rid of requestLegacyExternalStorage as well CAMERA permission from AndroidManifest.

the launchCamera / launchImageLibrary work without problems on all the devices (Android6-Android11), as far as if you actually used some kind option such as "save to storage", I am not sure.

**Reply by MayoudP** [reply]
Getting `> Configure project :react-native-image-picker
 WARNING: The following project options are deprecated and have been removed: ` after upgrading the lib, and cleaning again... ?
`error: package com.imagepicker does not exist`
